docs/diagrams/DoneCryptoSequenceDiagram.puml

```diff
@@ -0,0 +1,116 @@
+@startuml
+'https://plantuml.com/class-diagram
+!include Styles.puml
+
+box console COLOR_BRIGHT_GREEN
+participant ":InputParser" as InputParser COLOR_CONSOLE
+end box
+
+box command COLOR_YELLOW
+participant ":DoneCommand" as DoneCommand COLOR_COMMANDS
+end box
+
+box model COLOR_GREY
+participant ":InstrumentManager" as InstrumentManager COLOR_MODEL
+participant ":Instrument" as Instrument COLOR_MODEL
+end box
+
+box ui COLOR_LIGHT_BLUE
+participant ":TextUi" as TextUi COLOR_UI
+end box
+
+
+[-> InputParser : filterByCommandType({"done"}, instruments)
+activate InputParser
+
+InputParser-> DoneCommand : new DoneCommand()
+activate DoneCommand
+
+InputParser ->InputParser: getDoneInstrumentCommand(commandComponents[], instruments)
+activate InputParser
+
+InputParser ->InputParser: getIndex(commandComponents[])
+activate InputParser
+InputParser --> InputParser
+deactivate InputParser
+
+InputParser ->DoneCommand: setIndex(index)
```

Comment:
The setIndex is suppose to start a new activation bar outside of the Done constructor activation bar. It is not self-invocating